src/ucs/sys/event_set.c

```diff
@@ -73,6 +77,9 @@ ucs_status_t ucs_event_set_create(ucs_sys_event_set_t **event_set_p)
         goto err_free;
     }
 
+    event_set->max_events = ceil(UCS_ALLOCA_MAX_SIZE /
+                                 sizeof(struct epoll_event));
+
```

Comment:
Does `event_set->max_events` require?. It seems that this variable doesn't use in this file.

Reply:
@hiroyuki-sato Thank you, this is not needed anymore
will remove

Reply:
fixed